### PR TITLE
[Naitei][Feature][User] View bill list

### PR DESCRIPTION
**Pull request by TranMinhHuy2003**
## Related Tickets
[[User] Xem danh sách hóa đơn đã đặt hàng](https://edu-redmine.sun-asterisk.vn/issues/78825)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
bill_list_en
bill_list_vi
passrubocop_view_bill_list

### CODE REVIEW

**Comment by TranMinhHuy2003** [comment]
ready

File: app/views/cart/_cart_detail.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  </div>
+
+  <div>
+    <p class="row__price"><%= cart_detail.product.price %></p>
+  </div>
+  <div>
+    <p class="my-cart__row__quantity">
+      <%= form_with url: cart_add_path do |f| %>
+        <%= f.hidden_field :id, :value => cart_detail.product.id %>
+        <%= f.number_field :quantity, :value => cart_detail.quantity, :min => 0, style: "width: 40%;" %>
```
Comment (ghost) [inline app/views/cart/_cart_detail.html.erb]:
```suggestion
        <%= f.number_field :quantity, value: cart_detail.quantity, min: 0, style: "width: 40%;" %>
```

**Review by ghost — COMMENTED** [review]
cmt

**Comment by TranMinhHuy2003** [comment]
ready

File: app/views/bills/_bill.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<tr>
+  <td><%= bill_counter + Settings.counter_index %></td>
+  <td><%= l bill.created_at, format: :long %></td>
+  <td><%= send("#{I18n.locale}", bill.total_after_discount) %></td>
```
Comment (ghost) [inline app/views/bills/_bill.html.erb]: Dung public_send

**Review by ghost — COMMENTED** [review]
cmt

**Comment by TranMinhHuy2003** [comment]
ready

**Comment by TranMinhHuy2003** [comment]
ready

**Pull request merged**